app/src/main/java/com/example/campusguide/directions/Directions.kt

```diff
@@ -31,17 +37,12 @@ class Directions constructor(private var activity: AppCompatActivity){
         val request = JsonObjectRequest(Request.Method.GET, url, null,
```

Comment:
Why not use your VolleyRequest Class here?

Reply:
Will be used in the next PR